### PR TITLE
Sidebar ID Prop

### PR DESCRIPTION
**Pull request by joeflack4**
Allowed for an ID property on the sidebar element.

### CODE REVIEW

File: src/sidebar.js [hunk 1]
```diff
@@ -398,6 +404,7 @@ Sidebar.propTypes = {
 
 Sidebar.defaultProps = {
   docked: false,
+  sidebarId: '',
```
Comment (markusenglund) [inline src/sidebar.js]: This will give the sidebar an empty id. I think it's best to not set a default prop for sidebarId at all.

**Comment by markusenglund** [comment]
I want to merge this change. Could you remove the defaultProp first? Also, `dist-modules` is not stored in source control anymore, so could you remove that whole file as well?

**Comment by rluiten** [comment]
I was just about to raise a pull request for this myself, full steam ahead.
My use case is that I need to supply the containerId to react-scroll to be able to scroll content.

**Comment by markusenglund** [comment]
Ok. I suppose @rluiten's PR is a bit more comprehensive so I'll merge that one instead.

**Comment by markusenglund** [comment]
This feature was added in #160 instead.

**Pull request closed without merging**